content/blog/2018-05-10-onboarding-social-weather.md

```diff
@@ -0,0 +1,698 @@
+--- 
+slug: onboarding-social-weather
+title: 'The social weather of rOpenSci onboarding system'
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+date: 2018-05-09
+preface: "This blog post is the third of a 3-post series about a data-driven overview of rOpenSci onboarding. Read the intro to the series [here](https://ropensci.org/blog/2018/04/26/a-satrday-ct-series/), the first post about data collection [here](https://ropensci.org/blog/2018/04/26/rectangling-onboarding/), the second post about work quantification [here](https://ropensci.org/blog/2018/05/03/onboarding-is-work/)."
+categories: blog
+topicid: 
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- tidytext
+- sentimentr
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+Our [onboarding process](https://github.com/ropensci/onboarding/)
+ensures that packages contributed by the community undergo a
+transparent, constructive, non adversarial and open review process.
+Before even submitting my first R package to rOpenSci onboarding system
+in December 2015, I spent a fair amount of time reading through previous
+issue threads in order to assess whether onboarding was a friendly place
+for me: a newbie, very motivated to learn more but a newbie nonetheless.
+I soon got the feeling that yes, onboarding would help me make my
+package better without ever making me feel inadequate.
+
+More recently, I read [Anne Gentle’s
+essay](https://github.com/Open-Advice/Open-Advice/blob/c3e03b669f7ea7b20ef98378210cdbc8a7ea7edc/documentationsupport/AnneGentle.tex)
+in Open Advice where she mentions the concept of the *social weather* of
+an online community. By listening before I even posted, I was simply
+trying to get a good idea of the social weather of onboarding – as a
+side-note, how wonderful is it that onboarding’s being open makes it
+possible to listen?!
+
+In the meantime, I’ve now only submitted and reviewed a few packages but
+also become an [associate
+editor](https://github.com/ropensci/onboarding/#associate-editors). In
+general, when one of us editors talks about onboarding, we like to use
+examples illustrating the system in a good light. Quotes from thankful
+package authors, of amazed reviewers, etc. Would there be a more
+quantitative way for us to support our promotion efforts? In this blog
+post, I shall explore how a [tidytext](https://www.tidytextmining.com/)
+analysis of review processes (via their GitHub threads) might help us
+characterize the social weather of onboarding.
+
+### Preparing the data
+
+#### A note about text mining
+
+In this blog post, I’ll leverage the `tidytext` package, with the help
+of its accompanying book [“Tidy text
+mining”](https://www.tidytextmining.com/). Interestingly, the authors
```

Comment:
Does it mean you're pushing a bit or that I've now become a better author so that you feel free to make more advanced comments? I'm choosing the latter answer 😁